lib/util.c

```diff
@@ -607,7 +607,13 @@ static int _copyfile_helper(const char *from, const char *to, int flags)
         struct timeval tv[2];
         TIMESPEC_TO_TIMEVAL(&tv[0], &sbuf.st_atim);
         TIMESPEC_TO_TIMEVAL(&tv[1], &sbuf.st_mtim);
+#if defined(HAVE_FUTIMES)
         if (futimes(destfd, tv)) {
+#elif defined(HAVE_FUTIMESAT)
+	if (futimesat(destfd, NULL, tv)) {
+#else
+	if (utimes(to, tv)) {
```

Comment:
These lines have tab characters in them, can you please convert those to 8-spaces to match the surrounding style?  Thanks.